app/Http/Resources/V1/BuildApiResource.php

```diff
@@ -44,6 +45,7 @@ public function toArray($request)
                 $buildData['installations_percent'] = $this->applicationActiveDevices > 0 ?
                     (int) floor(($buildData['installations'] / $this->applicationActiveDevices) * 100) :
                     0;
+				$buildData['download_information'] = new DownloadResource($build);
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏This is going to redirect you always to the latest build and not the specific one you are trying to download: https://github.com/everli/uppy/blob/develop/app/Http/Controllers/Web/ApplicationController.php#L54
Also I think passing the whole download resource is overkill for just a download link 🤔 , so what about just setting a `download_url` ?

Reply:
Fixed in https://github.com/everli/uppy/pull/43/commits/425cbe6c4114212e960250fa39bce3bb8d1494f6